libs/ui/src/pure/Popover/PopoverMod.tsx

```diff
@@ -119,13 +145,7 @@ export default function Popover({
     update && update()
   }, [update])
 
-  useEffect(() => {
-    if (!show) return
-
-    const interval = setInterval(updateCallback, 100)
-
-    return () => clearInterval(interval)
-  }, [])
+  useInterval(updateCallback, show ? 100 : null)
```

Comment:
thanks for fixing, but why do popper need a interval in the first place? https://popper.js.org/react-popper/v2/
at least doesn't look like the example they provide

Reply:
It's very legacy thing :) 
Probably we should use https://reach.tech/tooltip/ unstead

Reply:
👍